spring-rabbit/src/main/java/org/springframework/amqp/rabbit/config/RetryInterceptorBuilder.java

```diff
@@ -200,9 +200,6 @@ protected void applyCommonSettings(AbstractRetryOperationsInterceptorFactoryBean
 	public abstract T build();
 
 
-	private RetryInterceptorBuilder() {
-	}
```

Comment:
??? I don't see package protected replacement.
Intentional? 
`protected` class is compiled with `protected` ctor?